mod_ci/controllers.py

```diff
@@ -554,7 +554,17 @@ def start_ci():
                 g.log.debug(payload)
 
         elif event == "pull_request":
-            # If it's a PR, run the tests
+            # Check if user blacklisted
+            if BlockedUsers.query.filter(BlockedUsers.userID == payload[
+                   'pull_request']['user']['id']).first():
+                log.critical("User Blacklisted")
+                gh_commit.post(
+                    state=Status.ERROR,
+                    description="CI start aborted. You may be blocked from accesing this functionality",
+                    target_url=url_for(@mod_ci.route('/'), _external=True)
```

Comment:
No, that's not the correct way to link.

See https://github.com/CCExtractor/sample-platform/blob/8403bae4583da989a96633ef6c05844ae6e5ecb1/mod_auth/controllers.py#L53